ui/lib/core/addon/utils/client-count-utils.js

```diff
@@ -152,7 +131,6 @@ export const namespaceArrayToObject = (totalClientsByNamespace, newClientsByName
   const nestNewClientsWithinNamespace = totalClientsByNamespace?.map((ns) => {
     const newNamespaceCounts = newClientsByNamespace?.find((n) => n.label === ns.label);
     if (newNamespaceCounts) {
-      const { label, clients, entity_clients, non_entity_clients, secret_syncs } = newNamespaceCounts;
```

Comment:
Cleaned up instances of destructuring so adding future client types is less cumbersome